patterns/template-search-portfolio.php

```diff
@@ -1,6 +1,6 @@
 <?php
 /**
- * Title: Portfolio Search
+ * Title: Portfolio search template
```

Comment:
did we not say that appending "template" was redundant because the modal states clearly that we are talking about templates?

Reply:
I did yea, but after thinking it through and seeing it all in effect, I think it's fine appending template.